llvm/utils/TableGen/RegisterInfoEmitter.cpp

```diff
@@ -1286,8 +1286,8 @@ void RegisterInfoEmitter::runTargetDesc(raw_ostream &OS) {
     }
     OS << "};\n";
 
-    OS << "\nstatic const TargetRegisterClass *const "
-       << "NullRegClasses[] = { nullptr };\n\n";
+    OS << "\nstatic const unsigned "
```

Comment:
Could use uint16_t to save space? All in-tree targets have well under 1000 classes currently.

Reply:
Probably, but I'd prefer not to make such a functional change mixed in with this PR.